pkg/kudoctl/cmd/init_test.go

```diff
@@ -212,6 +208,9 @@ func TestNewInitCmd(t *testing.T) {
 		{name: "name and version together invalid", flags: map[string]string{"kudo-image": "foo", "version": "bar"}, errorMessage: "specify either 'kudo-image' or 'version', not both"},
 		{name: "crd-only and wait together invalid", flags: map[string]string{"crd-only": "true", "wait": "true"}, errorMessage: "wait is not allowed with crd-only"},
 		{name: "wait-timeout invalid without wait", flags: map[string]string{"wait-timeout": "400"}, errorMessage: "wait-timeout is only useful when using the flag '--wait'"},
+		{name: "crd-only and upgrade together invalid", flags: map[string]string{"crd-only": "true", "upgrade": "true"}, errorMessage: "'--upgrade' and '--crd-only' can not be used at the same time"},
```

Comment:
I know you didn't start it, but not all of these invalid options make sense at first glance (at least not for me). wdyt about explaining why these combinations are invalid? e.g.:
```suggestion
		{name: "crd-only and upgrade flags together are invalid: you can't upgrade *only* crds", flags: map[string]string{"crd-only": "true", "upgrade": "true"}, errorMessage: "'--upgrade' and '--crd-only' can not be used at the same time"},
```